docs/SDL/OnAllowSDLFunctionality/index.md

```diff
@@ -1,17 +1,79 @@
 
 ### Notification
+Notifies about user/HMI allowing SDL functionality or disallowing access to all mobile apps on the specified device.
+
+!!! MUST  
+Send `OnAllowSDLFuntionality` notification to SDL as a result of user’s choice about device data consent via pop-up dialog or user’s settings change in Settings Menu (if applicable).
+!!!
+
+!!! NOTE   
+
+1. SDL ignores all invalid notifications which come from HMI (invalid JSON, invalid data types/bounds etc).   
+2. In case SDL PoliciesManager receives _SDL.OnAllowSDLFunctionality_ with _allowed=false_ and without _device_ param from HMI, PoliciesManager must record all of currently registered devices as NOT consented in Local PT.   
+3. In case PoliciesManager receives _SDL.OnAllowSDLFunctionality_ with _allowed=true_ and without _device_ param from HMI, PoliciesManager must:   
+    - record all of currently registered devices as consented in Local PT;   
+    - send _BC.ActivateApp_(params, level: ”default_hmi”-value-from-assigned-policies) to HMI (even if this “default_hmi” value is NONE or BACKGROUND).   
+    - notify mobile application via _OnHMIStatus_(params, level: ”default_hmi”-value-from-assigned-policies).   
+4. In case PoliciesManager receives _SDL.OnAllowSDLFunctionality_ with _allowed=false_ and with _device_ param, PoliciesManager must record the named device (_device_ param) as NOT consented in Local PT ("user_consent_records" -> _device_ sub-section) and send _BC.ActivateApp_ request with _level_ param of the value from 'default_hmi' key of 'pre-DataConsent'section of Local PT to HMI.   
+5. In case PoliciesManager receives _SDL.OnAllowSDLFunctionality_ with _allowed=true_ and with _device_ param from HMI, PoliciesManager must record the named device (_device_ param) as consented in Local PT ("device_data" section \<device_id> subsection "user_consent_records" -> "device" sub-section).
+
+!!!
 
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
+|Name|Type|Mandatory|Description|
 |:---|:---|:--------|:---------|
-|device|[Common.DeviceInfo](../../common/structs/#deviceinfo)|false||
-|allowed|Boolean|true||
-|source|[Common.ConsentSource](../../common/enums/#consentsource)|true||
+|device|[Common.DeviceInfo]|false|If no device is specified permission counts for SDL functionality in general.|
+|allowed|Boolean|true|Must be true if allowed|
+|source|[Common.ConsentSource]|true|-|
+
+[Common.DeviceInfo]: ../../common/structs/#deviceinfo
+[Common.ConsentSource]: ../../common/enums/#consentsource
+
+
+### Sequence Diagrams
+
+Preconditions:   
+- Device is connected to the HU.   
+- App_1 is running on this device and is registered with SDL.   
+- App_1 presents in the list of registered apps on HMI.   
+
+|||
+The User consents the device.
+![OnAllowSDLFunctionality](../activateapp/assets/User%20consents%20the%20device2.png)
+|||
+
+|||
+The User does NOT consent the device.
+![OnAllowSDLFunctionality](../activateapp/assets/User%20does%20not%20consent%20the%20device1.png)
+|||
```

Comment:
@DrachenkoAnastasiia Alright, I have no idea why these diagrams are not appearing on the site. There doesn't seem to be much consistency on the matter. The only thing that I am thinking may be the cause is that these are the only links that both:

1. Reference an external assets directory (`activateapp`) AND
2. Contain spaces in their directory (`%20`)

All link which have one or the other appear to work, and the link to these diagrams work in the ActivateApp documentation page.

I do not know why this combination would cause a problem such as this, but it is the only explanation that I see at the moment.